app/models/companies_house_caller.rb

```diff
@@ -0,0 +1,45 @@
+class CompaniesHouseCaller
```

Comment:
I know this is where this logic has sat in other projects, but I've never been comfortable with it. IMHO I think a model should represent something that persists, and so far we've followed this pattern.

Therefore I'd like to suggest moving this to being a service object, or into the `lib/` folder.

Reply:
It felt a bit funny to me as well so I'm happy to move it 👍